app/views/layouts/mailer/mailer_layout.html.slim

```diff
@@ -0,0 +1,15 @@
+doctype html
+html
+  head
+    meta content="text/html; charset=UTF-8" http-equiv="Content-Type"
+
+  body
+      p test line
+
+      == yield
+
+
+  div id="footer"
```

Comment:
the footer should be nested under the body (just add one tab to each line). consider using that footer as your test line, though.